wee_slack.py

```diff
@@ -3867,6 +3908,10 @@ class PluginConfig(object):
             desc='When activity occurs on a buffer, unhide it even if it was'
             ' previously hidden (whether by the user or by the'
             ' distracting_channels setting).'),
+        'auto_open_threads': Setting(
```

Comment:
Can you place this alphabetically in the list?

Reply:
Done